lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
+    attribute :priority,        Integer, default: 0
+    attribute :unit,            String
+    attribute :user_value,      Float
+
+    validates_presence_of :query
+
+    def self.collection_from_hash(init_hash)
+      init_hash.map do |key, user_value|
+        input = find(key)
```

Comment:
Maybe. But raising is a bit harsh here. I would simply ignore it, as in Scenario#inputs_present